README.md

```diff
@@ -167,6 +168,44 @@ If you want to create Visual Studio build files, e.g., if not using `ninja`, be
 
 You can cross compile liboqs for various platforms. Detailed information is available [in the Wiki](https://github.com/open-quantum-safe/liboqs/wiki/Platform-specific-notes-for-building-liboqs#cross-compiling).
 
+### Nix Flake
+
+Nix is a tool that makes builds and developer environments deterministic. With just Nix installed, you can install `liboqs` using one command.
+
+Need to install Nix? Consider using the [Lix installer](https://git.lix.systems/lix-project/lix-installer), a modern, community-focused implementation of the Nix tool, available for Linux and MacOS.
```

Comment:
Why don't we use the one from Determinate system? This one is not well known.

Reply:
I'm not overly opposed to using the Determinate Systems one. My logic is that lix is implemented in a memory safe language and has knowledgeable developers that I know.

I personally use the Lix implementation, and since I'm planning on maintaining the flake, it seemed reasonable to me to choose what I use myself. That said, it shouldn't really be any different from the Determinate Systems installer.

Reply:
> I'm not overly opposed to using the Determinate Systems one. My logic is that lix is implemented in a memory safe language and has knowledgeable developers that I know.
Hmm, Determinate system also uses memory safe language. What make Lix different than Determinate? 

I still insist on using the Determinate System installer. I don't want to apply number of stars, forks metric here, I guess it will not be just me wonder why `liboqs` made this decision, other users/maintainers as well.

Reply:
For what it's worth I use the Determinate system installer too although I don't know if it's the community standard or not. Both Lix and Determinate Systems come across as rather opinionated and I don't think we want liboqs taking a stance either way. Since the flake ought to with any nix installation that supports flakes, how about we leave nix installation instructions out of the README?